vioscsi/trace.h

```diff
@@ -83,13 +110,21 @@ extern int nVioscsiDebugLevel;
 #define VioScsiDbgBreak()
 #endif
 
-#define ENTER_FN() RhelDbgPrint(TRACE_LEVEL_VERBOSE, " --> %s.\n",__FUNCTION__)
+#define ENTER_FN()  RhelDbgPrint(TRACE_LEVEL_VERBOSE, " --> %s.\n",__FUNCTION__)
```

Comment:
Do we need this extra space before RhelDbgPrint here?

Reply:
Technically, no.
In similar vein, I missed an extra indent from the definition for ENTER_INL_FN() at L115...

Reply:
no worries, I'm not going to lose sleep over it, just stuck out as I was reading through. Otherwise, looks great, thanks for your continued contributions to the quality of this code!